### ISSUE TITLE
Biokepi.Metadata.git_commit is None when pinned

### ISSUE THREAD

**Issue by ihodes** [hammerlab/biokepi#218]
Unsure if it's None or actually holds the version when it's opam installed.

**Comment by smondet** [comment]
Yes I think opam-pin gets rid of the `.git` directory, so `git_commit` gets set to `None`.
